examples/worlds/sensors.sdf

```diff
@@ -136,21 +136,20 @@
             </vertical_velocity>
           </altimeter>
         </sensor>
-        <sensor name="air_pressure" type="air_pressure">
+        <sensor name="air_speed" type="air_speed">
```

Comment:
should we add a new air_speed sensor (and keep the existing air_pressure sensor)?

Reply:
Do you also need to add the `gz-sim-air-speed-system`?